packages/ramda-extension/src/between.js

```diff
@@ -10,17 +10,16 @@ import { curry } from 'ramda';
  * @return {Boolean}
  * @example
  *
- *      R_.between(4, 1, 5); //=> true
- *      R_.between(2.1, 3, 8); //=> false
- *      R_.between(100.1, 100.1, 102); //=> true
- *      R_.between(100.1)(100.1, 102); //=> true
+ *      R_.between(1, 5, 4); //=> true
+ *      R_.between(3, 8, 2.1); //=> false
+ *      R_.between(100.1, 102, 100.1); //=> true
 
 
  */
 
 
-const between = curry((val, a, b) => {
-	if (val >= a && val <= b) {
+const between = curry((min, max, val) => {
```

Comment:
This is not exactly pointfree implementation. Do you need help?
This is snippet mentioned before:
```js
const between = useWith(compose(allPass, R_.argumentsToList), [lte, gte])

console.log(
    between(1, 3)(0), // false
    between(1, 3)(4), // false
    between(1, 3)(2)  // true
)
```